default/python/AI/colonization/colony_score.py

```diff
@@ -1,15 +1,30 @@
+import freeOrionAIInterface as fo
 from logging import debug
 
 from common.option_tools import check_bool, get_option_dict
+from freeorion_tools.caching import cache_for_session
 
-# minimum evaluation score that a planet must reach so it is considered for outposting or colonizing
-MINIMUM_COLONY_SCORE = 60
-# resource production is evaluated as production * priority_for_the_resource * RESOURCE_PRIORITY_MULTIPLIER
-RESOURCE_PRIORITY_MULTIPLIER = 0.5
+# Minimum evaluation score that a planet must reach to it is considered for outposting or colonizing.
+MINIMUM_COLONY_SCORE = 600
+# Resource production is evaluated as production * priority_for_the_resource * RESOURCE_PRIORITY_MULTIPLIER.
+# So raising this makes it easier for planets to reach the minimum score and in relation reduces the value of
+# other scores, which are rated as factor * MINIMUM_COLONY_SCORE
+RESOURCE_PRIORITY_MULTIPLIER = 5
 
 DEBUG_COLONY_RATING = check_bool(get_option_dict().get("debug_planet_rating", "False"))
 
 
 def debug_rating(*args, **kwargs):
     if DEBUG_COLONY_RATING:
         debug(*args, **kwargs)
+
+
+@cache_for_session
+def use_new_rating() -> bool:
+    config_value = eval(get_option_dict().get("use_new_colony_rating", "1"))
```

Comment:
Eval sounds terrible.  Arbitrary Python code could be executed. 

I'd suggest to make check like that:  `get_option_dict().get("use_new_colony_rating", "false").lower() == 'true'`

Reply:
> Yeah, you can break the AI by writing a wrong config file

You could execute arbitrary code with the wrong config using eval, like rm the home folder of the user.  This is a security vulnerability. 

I am ok with int.